File: data/imported/knowledge/slides/java/merged/snippets/snippet_018.java
@Test
    void testAdjustmentProperties() {
        try (var pres = new Presentation()) {
            var conn = pres.getSlides().get(0).getShapes().addConnector(ShapeType.BENT_CONNECTOR3, 50, 50, 300, 200);
            if (conn.getAdjustments().size() > 0) {
                var adj = conn.getAdjustments().get(0);
                assertThat(adj.getName()).isNotNull();
                assertThat(adj.getRawValue()).isInstanceOf(Long.class);
                assertThat(adj.getAngleValue()).isInstanceOf(Double.class);
            }
        }
    }

File: data/imported/knowledge/slides/net/merged/snippets/snippet_018.cs
[Fact]
    public void TestAdjustmentProperties()
    {
        // Adjustment values expose name, raw_value, angle_value.
        using var pres = new Presentation();
        var conn = pres.Slides[0].Shapes!.AddConnector(ShapeType.BentConnector3, 50, 50, 300, 200);
        var adjustments = conn.Adjustments;
        if (adjustments is not null && adjustments.Count > 0)
        {
            var adj = adjustments[0];
            adj.Name.Should().NotBeNull();
            adj.RawValue.Should().BeOfType(typeof(int));
            adj.AngleValue.Should().BeOfType(typeof(float));
        }
    }